### PR TITLE
Updating streamers

### PR DESCRIPTION
**Pull request by Careyjmac**
Updating streamers to remove auth issue (jobs that take over an hour should be able to finish now) add aggregators (to be able to view the progress of the job in the dataflow console better) and lower the shard size (for now).

### CODE REVIEW

File: src/main/java/com/google/cloud/genomics/dataflow/readers/ReadStreamer.java [hunk 1]
```diff
@@ -77,11 +79,11 @@ public static StreamReadsRequest getReadRequests(String readGroupSetId) {
...
     return FluentIterable.from(contigs)
         .transformAndConcat(new Function<Contig, Iterable<Contig>>() {
-            @Override
-            public Iterable<Contig> apply(Contig contig) {
-              return contig.getShards(100000000L);
-            }
-          })
+          @Override
+          public Iterable<Contig> apply(Contig contig) {
+            return contig.getShards(5000000L);
```
Comment (pgrosu) [inline src/main/java/com/google/cloud/genomics/dataflow/readers/ReadStreamer.java]: I think Nicole and Dion previously discussed issues with hardcoding it, but maybe having a final constant and just manipulating that might be better.

File: src/main/java/com/google/cloud/genomics/dataflow/readers/VariantStreamer.java [hunk 2]
```diff
@@ -79,7 +81,7 @@ public static StreamVariantsRequest getVariantRequests(String variantSetId) {
         .transformAndConcat(new Function<Contig, Iterable<Contig>>() {
             @Override
             public Iterable<Contig> apply(Contig contig) {
-              return contig.getShards(100000000L);
+              return contig.getShards(5000000L);
```
Comment (pgrosu) [inline src/main/java/com/google/cloud/genomics/dataflow/readers/VariantStreamer.java]: See my previous comment regarding hardcoding it:

https://github.com/googlegenomics/dataflow-java/pull/106/files#r33701593

Comment (dionloy) [inline src/main/java/com/google/cloud/genomics/dataflow/readers/VariantStreamer.java]: +1 on moving it up to a constant. Add a TODO that it should be replaced
with a better heuristic

**Comment by Careyjmac** [comment]
PTAL

**Comment by dionloy** [comment]
LGTM once Travis is happy

**Comment by pgrosu** [comment]
Dion, what do you think about having it be public to play around with it, until we determine an algorithm for the optimal shard size?

**Comment by dionloy** [comment]
I think it's fine private now. Public constants are against our coding style, I'd rather not have people depend on it, and hopefully this is just very temporary while we figure something better out.

**Comment by pgrosu** [comment]
That's understandable - thanks.

**Comment by deflaux** [comment]
LGTM

**Comment by deflaux** [comment]
@Careyjmac merge this anytime you like

**Pull request merged**